### PR TITLE
Make running tests a smoother experience

### PR DESCRIPTION
**Pull request by jonathan-s**

### CODE REVIEW

File: djangocms_moderation/admin_actions.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 
 from cms.utils.urlutils import add_url_parameters
 
-from django_fsm import TransitionNotAllowed
 from djangocms_versioning.models import Version
 
+from django_fsm import TransitionNotAllowed
```
Comment (Aiky30) [inline djangocms_moderation/admin_actions.py]: This import seems to be out of order now. It should be above versioning.

File: tests/settings.py [hunk 2]
```diff
@@ -4,16 +4,14 @@
...
+        "aldryn_forms",
         "filer",
         "easy_thumbnails",
-        "absolute",
-        "aldryn_forms",
         "captcha",
-        "emailit",
+
         "djangocms_text_ckeditor",
-        "djangocms_versioning.test_utils.blogpost",
```
Comment (Aiky30) [inline tests/settings.py]: Should these versioning packages be removed?

File: tests/utils/factories.py [hunk 3]
```diff
@@ -4,20 +4,20 @@
...
 
-import factory
 from djangocms_versioning.models import Version
 from djangocms_versioning.test_utils.factories import (
     AbstractVersionFactory,
     PageVersionFactory,
 )
-from factory.fuzzy import FuzzyChoice, FuzzyInteger, FuzzyText
 
+import factory
```
Comment (Aiky30) [inline tests/utils/factories.py]: Factory should be at the top of the file. They were done wrong before. 

```
import string
import factory

from djangocms_versioning.models import Version
from djangocms_versioning.models import Version
from factory.fuzzy import FuzzyChoice, FuzzyInteger, FuzzyText

from .moderated_polls.models import Poll, PollContent, PollPlugin

```

File: tests/settings.py (hunk 2 above)
Comment (jonathan-s) [reply tests/settings.py]: Yes they are not necessary. Tests run fine without them, and these are really for internal use for djangocms-versioning tests.

File: tests/utils/factories.py (hunk 3 above)
Comment (jonathan-s) [reply tests/utils/factories.py]: isort would complain if I moved it to that location. So from what I can tell it follows the principle of grouping things according to stdlib, 3rd party and package imports.

File: djangocms_moderation/admin_actions.py (hunk 1 above)
Comment (jonathan-s) [reply djangocms_moderation/admin_actions.py]: Same here. Isort complains if I move it.

**Pull request merged**